friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+
+
+import '../App.css';
+
+class FriendForm extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            name: '',
+            age: '',
+            email: '',
+            friends: []
+        }
+    }
+
+    inputChangehandler = (e) => {
+        this.setState({
+            [e.target.name]: e.target.value
+        }, console.log(this.state))
+    }
+
+    onSubmit = (e) => {
+        e.preventDefault();
+        axios
+            .post(`http://localhost:5000/friends`, {
+                name: this.state.name,
+                age: this.state.age,
+                email: this.state.email
+            })
+            .then(response => {
+                this.props.addFriends(response.data);
+            })
+            .catch(err => console.log(err))
+        this.setState({
```

Comment:
You write really clean code, I thought you were using a formatter like Prettier or something.  You might want to look into some sort of code-formatter, you can make it format your code everytime you save, it's just a suggestion.